app/serializers/sidebar_section_serializer.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class SidebarSectionSerializer < ApplicationSerializer
+  attributes :id, :title, :links, :slug
+
+  def links
+    object.sidebar_section_links.map(&:linkable)
```

Comment:
Don't we need to serialize `linkable` here? I believe we only need the `name` and `value` attributes to be shipped down.

Reply:
We need "id" as well for update action. So actually we are shipping whole SidebarUrl